samples/unit-tests/pointer/members/demo.js

```diff
@@ -173,6 +180,48 @@ QUnit.test(
             0,
             'mousemove to 30px below 1.2: no unexpected events'
         );
+        // Clear to mark only failed tests.
+        events.length = 0;
+
+        // Bubble series (#18940)
+        const bigRadius = 60;
+        chart.addSeries({
+            type: 'bubble',
+            maxSize: bigRadius * 2,
+            minSize: 20,
+            data: [
+                [1, 9, 2],
+                [1.2, 9, 1],
+                [1.2, 6, 1]
+            ]
+        });
+        controller.setPosition(
+            chart.plotLeft + chart.series[2].points[0].plotX + bigRadius + 7,
+            chart.plotTop + chart.series[2].points[0].plotY
+        );
+        // Mouse move between the bubbles should trigger tooltip only for the
+        // first point at the start and the last point close to the end of the
+        // mouse move event due to searchKDTree including radius of the marker.
+        controller.moveTo(
+            chart.plotLeft + chart.series[2].points[0].plotX + bigRadius + 7,
+            chart.plotTop + chart.series[2].points[0].plotY + 70,
+            {},
+            true
```

Comment:
I am not sure if that flag works in headless tests, but I think we should remove drawing the mouse in tests?